packages/proxyIdentity/contracts/IdentityManager.sol

```diff
@@ -1,95 +1,114 @@
 pragma solidity 0.8.0;
 
+import "@openzeppelin/contracts/proxy/Clones.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
+import "./OfferableIdentity.sol";
+import "./IOfferable.sol";
+
 
 contract IdentityManager {
-  mapping(address => address) owners;
+  address libraryAddress;
+  address ethrRegistry;
+  address owner;
```

Comment:
I don't think we need it because it mostly adds logic intended to change owner that is not planned for Identity Manager

Reply:
If this is something that needs to be owned by EWF in the end, then it's better to have it transferrable.

Reply:
Since ethrRegistry now will be provided by `sendTransaction` (former `update`) caller and after discussion we decided to not allow to change identity library there is no need for IdentityManager owner any longer